sites/main-site/src/content/events/2025/SIG_metaomics_nftestfest.md

```diff
@@ -0,0 +1,46 @@
+---
+title: meta-omics nf-test fest
+subtitle: Hackathon for implementing nf-test in meta-omics related pipelines!
```

Comment:
```suggestion
subtitle: Hackathon to implement nf-test in meta-omics related pipelines!
```
Actually in the style docs: "Avoid gerund" (https://nf-co.re/docs/guidelines/documentation/writing_style).